### PR TITLE
Add BinaryAlexNet reference to docstring of exported function...

### PR DESCRIPTION
**Pull request by AdamHillier**
...so that the reference shows up in the Larq Zoo docs.

### CODE REVIEW

**Comment by AdamHillier** [comment]
(This is the only model in our docs that doesn't have a link to the corresponding paper, when listed on https://docs.larq.dev/zoo/api/literature.)

File: larq_zoo/literature/binary_alex_net.py [hunk 1]
```diff
@@ -156,6 +156,9 @@ def BinaryAlexNet(
 
     # Raises
         ValueError: in case of invalid argument for `weights`, or invalid input shape.
+        
+    # References
+        - [Binarized Neural Networks](https://papers.nips.cc/paper/6573-binarized-neural-networks)
```
Comment (AdamHillier) [inline larq_zoo/literature/binary_alex_net.py:161]:
```suggestion

    # References
        - [Binarized Neural Networks](https://papers.nips.cc/paper/6573-binarized-neural-networks)
```

**Review by jneeven — APPROVED** [review]

**Comment by AdamHillier** [comment]
@lgeiger what's the process to bump and re-deploy the docs?

**Comment by lgeiger** [comment]
> what's the process to bump and re-deploy the docs?

A new release will allows us to bump the requirement in the docs repo https://github.com/larq/zoo/pull/266

**Pull request merged**